octobot_trading/enums.py

```diff
@@ -31,8 +31,14 @@ class PositionSide(enum.Enum):
 class TradeOrderType(enum.Enum):
     LIMIT = "limit"
     MARKET = "market"
+    # stop loss == reduce only on futures
     STOP_LOSS = "stop_loss"
     STOP_LOSS_LIMIT = "stop_loss_limit"
+    # conditional orders
+    # TODO handle them
+    STOP_MARKET = "stop_market"
+    STOP_LIMIT = "stop_limit"
```

Comment:
I don't understand the difference between STOP_LIMIT and STOP_LOSS_LIMIT

Reply:
So those order types are only relevant for futures
STOP_LIMIT can be an order that opens or closes a position, when set to reduce only a STOP_LIMIT will be the same as an STOP_LOSS_LIMIT

So the STOP_LOSS_LIMIT will only close positions, while STOP_LIMIT can open or close positions

Reply:
same applies for STOP_MARKET / STOP_LOSS_MARKET

Reply:
should we name them CONDITIONAL_MARKET and CONDITIONAL_LIMIT ? That might be less confusing

Reply:
I renamed it to conditional